### ISSUE TITLE
Feature Request: load generated images in the jpg format.  

### ISSUE THREAD

**Issue by AbdullahAlfaraj** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#385]
This feature should fix. #383

**Comment by cdmusic2019** [comment]
Is it possible to support both formats at the same time? After all, the PNG format surpasses JPG in both quality and detail. Personally, I still prefer the PNG format.

**Comment by AbdullahAlfaraj** [comment]
@cdmusic2019 Absolutely! I’ll add an option for you to select your preferred format for saving images in the plugin output folders. PNG will be the default choice. Also, I’ll take care of the issue in #383 by making sure that JPG generated images are detected and loaded correctly.

**Comment by nekogori45** [comment]
> @cdmusic2019絶対に！プラグイン出力フォルダーに画像を保存するための優先形式を選択するオプションを追加します。PNG がデフォルトの選択になります。また、 JPG で生成された画像が正しく検出されロードされることを確認することで、#383の問題に対処します。

Are there any plans to support the webp format?
It seems that not only jpg-generated images, but also webp-generated images are not detected correctly.
It would be helpful if you could support the webp format as well.
